Build of C++ client on Ubuntu fails.

I tried to build "C++ Client" in REAME but an error occurred.
Please tell me how to deal with this error.

## Development Environment

```Shell
$ cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.4 LTS"

$ python3 --version
Python 3.8.10

$ pip --version
pip 20.0.2 from /usr/lib/python3/dist-packages/pip (python 3.8)

$ cmake --version
cmake version 3.16.3

$ gcc --version
gcc (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0

$ java --version​
openjdk 17.0.3 2022-04-19​

​$ ant -version​
Apache Ant(TM) version 1.10.7 compiled on October 24 2019
```

## Procedure

```
$ cd myapp
$ ls
CMakeLists.txt  api.yaml  client.cpp  services.zs  zswag/

$ mkdir build
$ cd build
$ cmake ..
$ cmake --build .
```

## Error message

```
...
...
[ 75%] Building CXX object CMakeFiles/myapp.dir/client.o
/home/m-kitahara/work/zswag/example/10_cpp/myapp/client.cpp:7:45: error: ‘MyService’ in namespace ‘myapp_services::services’ does not name a type
    7 | using MyService = myapp_services::services::MyService;
      |                                             ^~~~~~~~~
In file included from /home/m-kitahara/work/zswag/example/10_cpp/myapp/client.cpp:3:
/home/m-kitahara/work/zswag/example/10_cpp/myapp/build/myapp-zserio-cpp.zserio-gen/myapp_services/services/MyService.h:24:11: note: ‘myapp_services::services::MyService’ declared here
   24 | namespace MyService
      |           ^~~~~~~~~
/home/m-kitahara/work/zswag/example/10_cpp/myapp/client.cpp: In function ‘int main(int, char**)’:
/home/m-kitahara/work/zswag/example/10_cpp/myapp/client.cpp:18:45: error: ‘specUrl’ was not declared in this scope
   18 |     auto openApiConfig = fetchOpenAPIConfig(specUrl, *httpClient);
      |                                             ^~~~~~~
/home/m-kitahara/work/zswag/example/10_cpp/myapp/client.cpp:26:28: error: ‘MyService’ has not been declared
   26 |     auto myServiceClient = MyService::Client(openApiClient);
      |                            ^~~~~~~~~
make[2]: *** [CMakeFiles/myapp.dir/build.make:63: CMakeFiles/myapp.dir/client.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:497: CMakeFiles/myapp.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
```

--- a/README.md
+++ b/README.md
@@ -491,7 +491,7 @@ which contains the code to actually use the zswag C++ client.
 
 using namespace zswagcl;
 using namespace httpcl;
-using MyService = myapp_services::services::MyService;
+namespace MyService = myapp_services::services::MyService;
 
 int main (int argc, char* argv[])
 {
@@ -502,7 +502,7 @@ int main (int argc, char* argv[])
     auto httpClient = std::make_unique<HttpLibHttpClient>();
     
     // Fetch the OpenAPI configuration using the HTTP client
-    auto openApiConfig = fetchOpenAPIConfig(specUrl, *httpClient);
+    auto openApiConfig = fetchOpenAPIConfig(openApiUrl, *httpClient);
     
     // Create a Zserio reflection-based OpenAPI client that
     // uses the OpenAPI configuration we just retrieved.

